src/app.jsx

```diff
@@ -178,7 +179,7 @@ export const Application = () => {
         if (currentPath === "")
             return;
 
-        setSelected(sel);
+        setSelected({ type: "current", data: { name: sel } });
```

Comment:
I would say you can adjust the `selected` state variable to be array instead of instroducing the 'type' property. The array can contain one or multiiple elements, and according to the lenght of the selected array some of the actions will be hidden.